### ISSUE TITLE
Planner Plan creation

### ISSUE THREAD

**Issue by alexc-MSFT** [pnp/provision-assist-m365#24]
Functionality to enable creation of Planner 'Plans' (when available in MS Graph APIs).

**Comment by wapster182** [comment]
Hey @alexc-MSFT, this would be a great enhancement for our users to create the simple Planner Plan as a collab space. Can you add this to the collab space types? It would be great.

**Comment by alexc-MSFT** [comment]
Hey @wapster182, so I've revisited the graph API endpoints today and we now seem to have planner available. I've got a few questions around how you see this working as the graph API supports creation of plans in new and existing groups.

- Should we have it so that when you select planner on the collab space types screen it asks you if you want to attach to an existing group? (attempt to pull a list of groups they are a member of) or if you want to create a new group?
- Or we abstract the word 'group' from it all and simply create a new group for them when they create the plan.

I think if they've created a group anyway they can go and add a planner plan without needing to use the tool anyway.

**Comment by wapster182** [comment]
Yes, I am with you and your second way of implementation. Planner would be a new "group" or better Collaboration Space type where it creates a new group with a planner.

Another way in my head would be to make the Power App check and ask for what the user wants, like if you selected the Planner as the type, in the next step it could ask "Do you want to create a new planner or add it to one of your following groups" and then list the ones the user has. What do you think?

**Comment by alexc-MSFT** [comment]
@wapster182 Thanks, glad we are aligned on the implementation in terms of creating a 'new' planner plan/group.

On the adding it to an existing group, I think we can just have a message to say the request is for new planner plans where you don't have a group rather than try and build functionality to create the plan and add to a group as after all they can do that themselves as far as I am aware? I will double check.

Thanks
